devel/git-annex/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           haskell-stack 1.0
+
+name                git-annex
+version             7.20190819
+checksums           rmd160  596f70e63d93b37f449869a84ff6677e9a4e8af4 \
+                    sha256  9e794baf81f3fcc0359ec9c0f22f5d5cad1ea9446958e53acafe747c48ef7ebb \
+                    size    1253032
+
+description         git-annex allows managing files with git, without checking the file contents into git
+long_description    \
+	git-annex allows managing files with git, without checking the file \
+	contents into git. While that may seem paradoxical, it is useful when \
+	dealing with files larger than git can currently easily handle, whether due \
+	to limitations in memory, time, or disk space.
+license             AGPL-3
+homepage            https://git-annex.branchable.com/
+
+maintainers         nomaintainer
+categories          devel haskell
+platforms           macosx
+
+master_sites        https://hackage.haskell.org/package/${name}-${version}
+
+# The downloaded copy of GHC assumes utimensat(2) is available, which is not
+# true for older systems
+depends_build-append \
+                    port:ghc
+configure.args-append \
+                    --system-ghc
+build.args-append   --system-ghc
+destroot.args-append \
+                    --system-ghc
```

Comment:
May I suggest something in the spirit of
```
haskell.use_system_ghc yes
```
instead?

Reply:
I added this in https://github.com/macports/macports-ports/pull/4632

Reply:
Whoops, not this, but @neverpanic’s code above:
```
configure.library_path
configure.cpath
```

It works.